doc/tutorial.rst

```diff
@@ -118,7 +118,7 @@ entirety:
   13              encoded = encoded + ' '
   14          else:
   15              x = letters.index(letter) + shift
-  16              encoded=encoded + letters[x]
+  16              encoded = encoded + letters[x]
```

Comment:
Why remove this?

Reply:
Pylint no longer does the bad-whitespace check (since https://github.com/PyCQA/pylint/issues/3572 / related PRs). I thought it would be better to not have mistakes in the example that would not be picked up.

Reply:
Ouups. My bad. You are totally right.